src/index.tsx

```diff
@@ -38,6 +38,32 @@ dbRef
   });
 
 const initializeCourseViewer = (): void => {
+
+  // Read the cookie and check whether the given user is authorized
+  if (getCookieValue('user') !== '') {
```

Comment:
```suggestion
  if (getCookieValue('user')) {
```
No need for !==, an empty string is considered falsy in JS. Can you tell me the return type for getCookieValue just to make sure?